### PR TITLE
FIX: Remove semicolon causing null statement in if statement

### PR DESCRIPTION
**Pull request by drewcoaustralia**
FIX: removes offending semicolon in an if statement in resources.cpp

### CODE REVIEW

File: coresdk/src/backend/graphics_driver.cpp [hunk 1]
```diff
@@ -1054,7 +1054,7 @@ void sk_fill_rect(sk_drawing_surface *surface, sk_color clr, float *data, int da
 //  Triangles
 //
 
-void sk_draw_triangle(sk_drawing_surface *surface, sk_color clr, float x1, float y1, float x2, float y2, float x3, float y3)
+void sk_draw_aa_triangle(sk_drawing_surface *surface, sk_color clr, float x1, float y1, float x2, float y2, float x3, float y3)
```
Comment (macite) [inline coresdk/src/backend/graphics_driver.cpp]: Triangles do not need to be axis aligned. So just sk_draw_triangle is fine.

File: coresdk/src/backend/graphics_driver.cpp [hunk 2]
```diff
@@ -1082,14 +1082,14 @@ void sk_draw_triangle(sk_drawing_surface *surface, sk_color clr, float x1, float
     }
 }
 
-void sk_fill_triangle(sk_drawing_surface *surface, sk_color clr, float *data, int data_sz)
+void sk_fill_aa_triangle(sk_drawing_surface *surface, sk_color clr, float x1, float y1, float x2, float y2, float x3, float y3)
```
Comment (macite) [inline coresdk/src/backend/graphics_driver.cpp]: Ditto.

**Comment by macite** [comment]
Looks good, if you can just change back to sk_draw_triangle. aa = axis aligned and only relates to rectangles -- meaning they can be represented as an x, y, width and height.

**Comment by drewcoaustralia** [comment]
Shall fix and repush

**Pull request closed without merging**